conf/core/ConfigOptions.asciidoc

```diff
@@ -1638,7 +1638,7 @@ for job output storage, the ID will be stored in the job status table of the ass
 === json.format.hootenanny
 
 * Data Type: bool
-* Default Value: `false`
+* Default Value: `true`
```

Comment:
Changing this default to `true` because in the constructor for `OsmJsonWriter` the value isn't read and is set to `true`.  So almost all of the tests rely on that default instead of this one.  The other cases that relied on this default being `false` have been updated to change the settings.